presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveCoercion.java

```diff
@@ -206,6 +238,42 @@ public void testHiveCoercionParquet()
         doTestHiveCoercion(HIVE_COERCION_PARQUET);
     }
 
+    @Requires(AvroRequirements.class)
+    @Test(groups = {HIVE_COERCION, HIVE_CONNECTOR})
+    public void testHiveCoercionAvro()
+            throws SQLException
+    {
+        doAvroTestHiveCoercion(HIVE_COERCION_AVRO);
+    }
+
+    private void doAvroTestHiveCoercion(HiveTableDefinition tableDefinition)
```

Comment:
Inline this method in the test. It makes sense to abstract the logic of `doTestHiveCoercion` because it is being called from multiple tests but this logic is only called from the Avro test.